tests/manual-test-cases/Group18-VIC-UI/setup-testbed.robot

```diff
@@ -207,7 +207,7 @@ Deploy Nimbus BrowserVm For NGC Testing
     [Return]  ${user}-${name}  ${ip}
 
 Deploy Nimbus ESXi Server For NGC Testing
```

Comment:
The main reason I decided to use my own keyword is that I need to use a different naming convention for the testbeds. For example, the name for a VCVA to be created should follow the following convention.

```
${name}=  Evaluate  'UITEST-VC%{TEST_VSPHERE_VER}-' + str(random.randint(1000,9999))  modules=random
```
On the other hand, the existing keyword from Nimbus-Util.robot doesn't capture my need.
```
${name}=  Evaluate  'VC-' + str(random.randint(1000,9999))  modules=random
```